docs/learn-transactions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: learn-transactions
+title: Transactions
+sidebar_label: Transactions
+---
+
+Transactions are used to send a balance to another user. To begin sending transactions, we will
+learn how to using Polkadot User Interface.
```

Comment:
"learn how to using" sounds weird to me. Maybe, change sentence to "We will learn how to begin sending transactions by using the Polkadot User Interface"